packages/legacy/core/App/localization/en/index.ts

```diff
@@ -460,6 +460,8 @@ const translation = {
     "ProofAccepted": "Proof Accepted",
     "ProofRejected": "Proof Rejected",
     "RequestedCredentialsCouldNotBeFound": "Requested credentials could not be found",
+    "PresentationMetadataNotFound": "Could not find presentation metadata",
+    "CredentialMetadataNotFound": "Could not find credential metadata",
```

Comment:
These feel maybe like internal error messages, not sure where these are shown. But if this happens there's not much the user can do about it right?

Reply:
No, this is just for consistency. We can also remove the translation for these errors if someone has a strong preference for it.